src/tests/reducers/articleReducer.test.js

```diff
@@ -254,6 +266,28 @@ describe('articlesReducer', () => {
       commentsPayload: {},
       likeDislikeSuccess: false,
       likeDislikeError: connectionError,
+      confirmDelete: false,
+    });
+  });
+
+  it('should set DELETE_ARTICLE_SUCCESS to true when article has been deleted', () => {
+    const action = {
+      type: DELETE_ARTICLE_SUCCESS,
+      payload: true,
+    };
+    const currentState = articlesReducer(initialState, action);
+    expect(currentState).toEqual({
```

Comment:
Everyone seems to be doing this which is making this file ambiguous. How about this:
```
expect(...).toEqual({
    ...initialState,
    confirmDelete: action.payload,
});
```
@mendozabree. If it works, advise the next victims.

Reply:
It has worked and I have refactored both the `articleReducer.test.js` and `userReducer.test.js` files for uniformity and good maintainability of the codebase